src/layouts/ContestLayout.js

```diff
@@ -102,6 +96,26 @@ const ContestLayout = (props) => {
                   Submit Finding
                 </Link>
               ) : null}
+              {props.data.markdownRemark &&
+              props.data.markdownRemark.frontmatter &&
+              !props.data.markdownRemark.frontmatter.altUrl ? (
+                <Link
+                  to={`/reports/${props.data.markdownRemark.frontmatter.slug}`}
+                  className="button cta-button button-medium primary"
+                >
+                  View Report
+                </Link>
+              ) : null}
+              {props.data.markdownRemark &&
+              props.data.markdownRemark.frontmatter &&
+              props.data.markdownRemark.frontmatter.altUrl ? (
+                <Link
+                  to={props.data.markdownRemark.frontmatter.altUrl}
+                  className="button cta-button button-medium primary"
+                >
+                  View Report
+                </Link>
+              ) : null}
```

Comment:
You can destructure `markdownRemark` similar to the approach with `contestsCsv`:
```
const { markdownRemark } = props.data;
```

It seems like markdownRemark could possibly be empty or undefined?
If so, you can define a variable to tell whether or not there is data for a report link (You can probably think of a better variable name for this):
```
const canViewReport = Boolean(markdownRemark.frontmatter)
```

```suggestion
              {canViewReport ? (
                <Link
                  to={markdownRemark.frontmatter.altUrl || `/reports/${markdownRemark.frontmatter.slug}`
                  className="button cta-button button-medium primary"
                >
                  View Report
                </Link>
              ) : null}
```